src/generator/WDL/entities/WorkflowGenerator.js

```diff
@@ -8,7 +8,14 @@ import { buildDeclarations } from '../utils/utils';
 
 export default class WorkflowGenerator {
   constructor(wfStep, settings) {
+    this.wfBodyElementGenerators = {
+      call: this.genCall,
+      scatter: this.genScatter,
+      whileloop: this.genWhile,
+      if: this.genIf,
+    };
```

Comment:
You converted `if_stmt` -> `if`, but `while_loop` -> `whileloop`... I still don't see the logic. I wish you would keep names clear and concise, no matter how ugly and complicated they are in hermes grammar =) Ok, go on, if you are sure.